### DISCUSSION TITLE
can't share the calendar

### DISCUSSION THREAD

**Discussion by Croustillan** [tutao/tutanota#6404] [Q&A]
I put the bug in the desktop app but it is everywhere it is about payment.

I pay for the revolutionary account but I can't share calendar with others.

**Comment by rezbyte** [comment]
Hi @Croustillan!

Sorry to hear you are having issues with sharing calendars. Please email customer support by using the support dialog inside the Tuta Mail client.

To use the support dialog, press 'Show Help' button (![Screenshot from 2024-01-12 12-12-47](https://github.com/tutao/tutanota/assets/48037381/842cb5c0-be02-47cd-85f7-8ef5801a9b4a)) on the toolbar on the bottom left corner of the screen, then select 'Support'. The support dialog should appear and then you must write your problem into the text box. After some articles appear, scroll to the very bottom of the list & click the button labelled 'Contact support'.

**Comment by Croustillan** [comment]
I tried the support before I came here, but I didn't scroll till the end, it had to many article :P...
Sorry and thanks for your response.

--
Christian Provost
Les Petits Fermiers





Jan 12, 2024, 06:18 by ***@***.***:
